flutterPractice03/main.dart

```diff
@@ -0,0 +1,8 @@
+int sumNumbers(int a, int b) {
+  return a+b;
```

Comment:
Please add a space around the `+` sign. Visual Studio Code can format this for you.